src/main/java/org/syncninja/util/TwoFilesComparison.java

```diff
@@ -0,0 +1,43 @@
+package org.syncninja.util;
+
+import java.io.BufferedReader;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.*;
+
+public class TwoFilesComparison {
+
+    public static ArrayList<String[]> compareFiles(String newFilePath, String oldFilePath) throws IOException {
```

Comment:
I believe the old file should be List of Lines right as we will be getting it from the StateTree